TLS/src/main/java/de/rub/nds/tlsattacker/dtls/protocol/handshake/ClientHelloDtlsMessage.java

```diff
@@ -22,10 +22,10 @@
 public class ClientHelloDtlsMessage extends ClientHelloMessage {
 
     @ModifiableVariableProperty(type = ModifiableVariableProperty.Type.COOKIE)
-    ModifiableByteArray cookie;
+    ModifiableByteArray cookie = null;
```

Comment:
FindBugs remarked an uninitialized read on the "UnsafeGet" operation later in the constructor code. Even if implicitly initialized to null I think it's cleaner to write this out here. Handling within the other code is not affected as far as I see.